test/test_wordcloud.py

```diff
@@ -229,6 +230,18 @@ def test_mask_contour():
     assert_true(all(sm_array[100, 300] == [0, 0, 255]))
 
 
+def test_mask_contour_transparent():
+    # test if mask contour can be drawn on a transparent background
+    # https://github.com/amueller/word_cloud/pull/389
+    examples_dir = os.path.realpath(os.path.join(os.path.dirname(__file__), '..', 'examples'))
+    mask_file = os.path.join(examples_dir, 'alice_mask.png')
+    mask = np.array(Image.open(mask_file))
+    w = WordCloud(mode='RGBA', background_color=None, contour_color='blue', contour_width=2, mask=mask,
+                  max_words=50, width=800, height=800)
+    w.generate(THIS)
+    w.to_file(NamedTemporaryFile(suffix=".png").name)
```

Comment:
Why is this necessary?

Reply:
That's how the previously buggy code path is triggered. If you don't write to a file the contour is not drawn. The exception happens here:
```
ret = np.array(img) * np.invert(contour)
ValueError: operands could not be broadcast together with shapes (900,900,4) (900,900,3)
```

Reply:
you could just do ``to_image()``.

Reply:
Ok. Fixed that.